mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -234,9 +234,18 @@ def delete_tsdb_resources(self):
                 drop_statements.append(
                     self.tables[table]._drop_subtable_query(subtable=subtable[0])
                 )
-            self.connection.run(
-                statements=drop_statements, timeout=self._timeout, retries=self._retries
-            )
+            try:
+                self.connection.run(
+                    statements=drop_statements,
+                    timeout=self._timeout,
+                    retries=self._retries,
+                )
+            except Exception as e:
+                logger.warning(
+                    "Failed to delete TDEngine resources, you may need to delete them manually",
```

Comment:
How will the user know what to delete though?

Reply:
It quite depends the error,  but I’ve improved the error message to guide the user to the specific resources - https://github.com/mlrun/mlrun/pull/6706/commits/51d6391be1dcf42e5aa902dad1fc1837c7a90d7d